research/utils/visdom_utils.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+import os
+import time
+import random
+
+import cv2
+from visdom import Visdom
+from visdom import server
+from torchvision.transforms import ToTensor
+from torch import Tensor
+import matplotlib.pyplot as plt
+import matplotlib
+import pdb
+import signal
+import hydra
+
+
+matplotlib.use("tkagg")
+
+
+@hydra.main(config_path="configs/visdom.yaml")
+def make_visdom(cfg):
```

Comment:
You might have 1 visdom configuration per experiment, so it is hardcoded now but should probably change that later